composer.json

```diff
@@ -11,9 +11,15 @@
     ],
     "require": {
         "php": "^8.1.0",
-        "guzzlehttp/guzzle": "^7.5.0"
+        "php-http/discovery": "^1.15",
+        "php-http/multipart-stream-builder": "^1.2",
+        "psr/http-client-implementation": "^1.0.1",
```

Comment:
psr/http-factory-implementation should also be listed to benefit fromPsr17Factory

Reply:
thanks for the hint